<launch>
  <param name="car_num" value="1" type="int"/>
  <node pkg="multi_car" name="multi_traj_node" type="multi_traj_node"  output="screen">
    <remap from="/car0/real_pose" to="/vrpn_client_node/car0/pose"/>
    <remap from="/car0/real_vel" to="/vrpn_client_node/car0/twist"/>
    <!-- <remap from="real_pose1" to="/vrpn_client_node/car1/pose"/>
    <remap from="real_vel1" to="/vrpn_client_node/car1/twist"/>
    <remap from="real_pose2" to="/vrpn_client_node/car2/pose"/>
    <remap from="real_vel2" to="/vrpn_client_node/car2/twist"/> -->
  </node>
  <node name="rvizvisualisation" pkg="rviz" type="rviz" output="log" args="-d $(find multi_car)/launch/test.rviz" />

</launch>